src/server/pegasus_mutation_duplicator.cpp

```diff
@@ -189,6 +211,9 @@ void pegasus_mutation_duplicator::on_duplicate_reply(uint64_t hash,
             // retry this rpc
             _inflights[hash].push_front(rpc);
             _env.schedule([hash, cb, this]() { send(hash, cb); }, 1_s);
```

Comment:
Will the non-idempotent request be retried twice here?

Reply:
Yes,it will. So I record specific `raw_key` into log.